build.gradle

```diff
@@ -138,6 +138,7 @@ project(':ambry-api') {
         compile "com.codahale.metrics:metrics-core:$metricsVersion"
         compile "org.codehaus.jackson:jackson-core-asl:$jacksonVersion"
         compile "org.codehaus.jackson:jackson-mapper-asl:$jacksonVersion"
+        compile "io.netty:netty-all:$nettyVersion"
```

Comment:
I wonder if we could limit the dependency to "io.netty:netty-buffer:$nettyVersion" in ambry-api. A lot of things (including java clients) depend on the ambry-api subproject.

Reply:
seems like a good idea, will update.